source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOHostConfig.mm

```diff
@@ -111,6 +111,14 @@ - (ACOResolverIFType)getResolverIFType:(NSString *)scheme;
     return [_resolvers getResolverIFType:scheme];
 }
 
+- (UIColor *)getHighlightColor:(ACRContainerStyle)style
+                     foregroundColor:(ForegroundColor)color
+                  subtleOption:(bool)isSubtle
```

Comment:
indenting looks off here

Reply:
thanks

---
In reply to: [272289790](https://github.com/Microsoft/AdaptiveCards/pull/2613#discussion_r272289790) [](ancestors = 272289790)